WMCO remove snippet from ifdef (#94037)
openshift/openshift-docs: modules/images-configuration-registry-mirror-configuring.adoc

ifdef::winc[]
[IMPORTANT]
====

> Windows images mirrored through `ImageDigestMirrorSet` and `ImageTagMirrorSet` objects have specific naming requirements. For information, see "Using Windows containers with a mirror registry".

Reviewer: Just wondering if there is an acceptable way to shorten this.
```suggestion
Windows images mirrored through `ImageDigestMirrorSet` and `ImageTagMirrorSet` objects have specific naming requirements, which you can find in "Using Windows containers with a mirror registry".
```

Author: > as described in "Using Windows containers with a mirror registry".  ??
